src/electron/views/report/unified-report-name-generator.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ReportNameGeneratorBuilder } from 'reports/report-name-generator-builder';
+
+export class UnifiedReportNameGenerator {
+    constructor(
+        private readonly reportNameGeneratorBuilder: ReportNameGeneratorBuilder = new ReportNameGeneratorBuilder(),
```

Comment:
nit: We usually avoid making the object itself responsible for creating things it's dependent on. In this case, I understand it's basically going to move this line to the initializers but we should try to remain consistent.

Reply:
The same applies to the other generator.

Reply:
This hasn't yet been resolved, so opening it again.

Reply:
it's just for setting the default value, since the class is kinda static, plus the constructor takes no parameters and needs no special initiating.
I'll close for now, we can update it in another PR tho